components/BannerArticles.js

```diff
@@ -0,0 +1,11 @@
+import BannerArticle from './BannerArticle';
```

Comment:
Maybe you can update the name of the file to BannerGroup? So that it's easier to read lang naman.

Reply:
n o